assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.hbs

```diff
@@ -1,3 +1,14 @@
 {{#if showUserLocation}}
-  <span class='map-location'>{{d-icon "map-marker"}} <span>{{user.custom_fields.geo_location.address}}</span></span>
+  {{mount-widget widget="user-location" args=user}}
 {{/if}}
+{{#if user.website_name}}
```

Comment:
@lumi-89 Generally, it's not very sustainable to replicate core markup in a plugin as it'll break every time the core logic changes. What's the argument for creating a new user website markup? Is it to make it easier to manage the location and website DOM elements? If so, I suggest it's best to do this via css and / or than re-creating the core markup, even if that makes it initially easier.

Reply:
@angusmcleod I agree. I was only following the approach used for hiding the default location element, but I should really have revised that part of the code. I have done that now.